ci/date-check/src/main.rs

```diff
@@ -36,29 +37,20 @@ impl fmt::Display for Date {
 }
 
 fn make_date_regex() -> Regex {
-    Regex::new(
-        r"(?x) # insignificant whitespace mode
-        <!--\s*
-        [dD]ate:\s*
-        (?P<y>\d{4}) # year
-        -
-        (?P<m>\d{2}) # month
-        \s*-->",
-    )
-    .unwrap()
+    Regex::new(r"[aA]s of (\w+) (\d{4})").unwrap()
```

Comment:
This shouldn't silently fail if newlines are used instead.

```suggestion
    Regex::new(r"[aA]s\s+of\s+(\w+)\s+(\d{4})").unwrap()
```